rosidl_generator_py/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ cmake_minimum_required(VERSION 2.8.3)
 
 project(rosidl_generator_py)
 
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+endif()
```

Comment:
Why would this be necessary? The package itself doesn't build any C++ code and each message generator needs to take care of its compiler flags on its own.